py/objint_longlong.c

```diff
@@ -279,6 +279,11 @@ mp_int_t mp_obj_int_get_checked(mp_const_obj_t self_in) {
     return mp_obj_int_get_truncated(self_in);
 }
 
+mp_uint_t mp_obj_int_get_uint_checked(mp_const_obj_t self_in) {
+    // TODO: Check overflow
```

Comment:
Same remark as above. Bonus: you wouldn't have to repeat this comment.

Reply:
Same reply, as well :)